### DISCUSSION TITLE
USB Ethernet & WiFi

### DISCUSSION THREAD

**Discussion by UncleJohnsBnd** [gtxaspec/wz_mini_hacks#173] [Q&A]
Does a Wi-Fi signal need to be available for the USB network adapter to be able to work?

I setup a brand new Cam3 with no SD card and got it working on the app.

Formatted 256gb SD to Fat32 and copied all the files in SD_ROOT over to the SD.  Which is one folder(ad everything in that folder) and one file.

Edited the wz_mini_config with NotePad++
ENABLE_USB_ETH="true"

Turned off camera

Turned off my 2.4 Wi-Fi

Put SD card in Wyze and started it up.  It just blinked for connection.... nothing on network adapter.

Turned off camera

Turned 2.4 Wi-Fi back on

Turned on camera

Camera connects

USB network adapter shows connected and data

Turn off 2.4 wifi and USB network card remains connected and everything runs

Just seems I can't get the usb network card to connect without there initally being wifi available.  Is that to be expected?

**Comment by gtxaspec** [comment]
this is a bug.  the system should only detect that the wireless driver is present, and not need it to establish a connection.  I will fix this. #174

**Reply by UncleJohnsBnd** [reply]
Awesome....  Let me know when updated and I will test it out.

This project is an awesome add on to Wyze cameras for sure!  Thanks for all the hard work and effort!

**Reply by UncleJohnsBnd** [reply]
Do you want me to open this as an Issue?

**Reply by gtxaspec** [reply]
already did =D #174
